### PR TITLE
Address pending comment from SECDIR review

### PR DESCRIPTION
**Pull request by tireddy2**

### CODE REVIEW

File: draft-ietf-dnsop-structured-dns-error.md [hunk 1]
```diff
@@ -503,7 +503,7 @@ purpose and the client can only display the resolver hostname that
 blocked the domain, error description for the EDE code and the
 suberror description for the "s" field to the end-user.
 
-When displaying the free-form text of "j" and "o", the browser SHOULD
+When displaying the free-form text of "j" and "o", the browser MUST
```
Comment (boucadair) [inline draft-ietf-dnsop-structured-dns-error.md:506]: The exception I had in mind is when explicit trusted resolvers are configured by an end user. 

BTW, I think the text should cover "c" as well, given that the attack we describe in the text right before uses "c". No?

Comment (tireddy2) [reply draft-ietf-dnsop-structured-dns-error.md:506]: Even if it is explicitly trusted by the end-user, "j" and "o" fields need not be clickable. The information in "c" field can be used to make a VOIP call, hence it is clickable but only with trusted revolvers.

Comment (boucadair) [reply draft-ietf-dnsop-structured-dns-error.md:506]: The text I was referring to is:

   For example, an end user can use the details
   in the "c" field to contact an attacker to solve the problem of being
   unable to reach a domain.  The attacker can mislead the end user to
   install malware or spyware to compromise the device security posture
   or mislead the end user to reveal personal data.

> The information in "c" field can be used to make a VOIP call, hence it is clickable but only with trusted revolvers.

Works for me. We don't have that explicitly mentioned in the text. Can you please update the PR to add a sentence for "c"? Thanks.

Comment (tireddy2) [reply draft-ietf-dnsop-structured-dns-error.md:506]: It is already covered by the preceding text: 

A client might choose to display the information in the "c", "j", and "o" fields if and only if the encrypted resolver has sufficient reputation, according to some local policy (e.g., user configuration, administrative configuration, or a built-in list of respectable resolvers). This limits the ability of a malicious encrypted resolver
to cause harm.

Comment (boucadair) [reply draft-ietf-dnsop-structured-dns-error.md:506]: This is not the same point. This does not cover the rendering part/clickable or not.

Comment (tireddy2) [reply draft-ietf-dnsop-structured-dns-error.md:506]: I will add the following text:
The contact details of "c" can be made into clickable links to provide a convenient way for users to initiate voice calls.

**Review by boucadair — APPROVED** [review]

**Pull request merged**